src/components/ebay-menu/test/test.browser.js

```diff
@@ -186,6 +188,25 @@ describe('given the menu is in the expanded state', () => {
         });
     });
 
+    // these two tests are failing on CI.
```

Comment:
Ah, I forgot to go and fix this for the CI. I might not get to this today, so if we are okay leaving this in for now I will fix as part of v2.4.0 (promise!) - which should also go out this week.

Reply:
I'm okay if you wait to fix this in 2.4.